src/actions/importing/abstract-importing-action.php

```diff
@@ -31,13 +33,35 @@ abstract class Abstract_Importing_Action implements Importing_Action_Interface {
 	 */
 	protected $options;
 
+	/**
+	 * The replacevar handler.
+	 *
+	 * @var Aioseo_Replacevar_Handler
+	 */
+	protected $replacevar_handler;
+
+	/**
+	 * The robots service.
+	 *
+	 * @var Aioseo_Robots_Service
+	 */
+	protected $robots;
+
 	/**
 	 * Abstract_Importing_Action constructor.
 	 *
-	 * @param Options_Helper $options The options helper.
+	 * @param Options_Helper            $options            The options helper.
+	 * @param Aioseo_Replacevar_Handler $replacevar_handler The replacevar handler.
+	 * @param Aioseo_Robots_Service     $robots             The robots service.
 	 */
-	public function __construct( Options_Helper $options ) {
-		$this->options = $options;
+	public function __construct(
+		Options_Helper $options,
+		Aioseo_Replacevar_Handler $replacevar_handler,
```

Comment:
The `Abstract_Importing_Action` seems to be intended for a broader use than AIOSEO.
Have you considered creating a `Replacevar_Translator` or `Replacevar_Transformer` interface and use that in the abstract class instead?
Although that assumes that more plugins we build importing actions for have a similar concept to replacevars, which I simply don't know if that is true.

Reply:
Although not introduced in this PR, my personal opinion is that classes with a `_Handler` or `_Manager` suffix are usually a bit vague about their responsibility. I often consider it a code smell and I expect to find a large class that does a lot of different things. Luckly this is not the case for the `Aioseo_Replacevar_Handler`, as it maps AIOSEO replacevars to Yoast replacevars and nothing more😄 . Though, I believe it can use a bit more specific name.
I'll leave it up to you _if_ you agree and want to address this and if so, if you want to do so in this PR.

Reply:
Yeah, we have a task that will deal with refactoring the structure there and indeed the naming of that class has been brought up for that, so we're dealing with it with a different PR. Nice catch!